### DISCUSSION TITLE
CM4 Heatsink

### DISCUSSION THREAD

**Discussion by maydaym3** [ByteWelder/Decktility#5] [General]
Hi, there just happen to be no photos of the build that include the underside of the PAD5 and the CM4, what heatsink did you use for this project? I have a pretty thick one, that I'm sure would increase the thickness of the display section quite a bit.

**Comment by KenVanHoeylandt** [comment]
Hi! There was no heatsink used. The fan's airstream direction goes towards the CM4, so I thought it was sufficient for my purposes.
